runelite-client/src/main/java/net/runelite/client/plugins/bosstimer/Boss.java

```diff
@@ -56,7 +56,8 @@
 	THERMONUCLEAR_SMOKE_DEVIL(NpcID.THERMONUCLEAR_SMOKE_DEVIL, 8400, ChronoUnit.MILLIS, ItemID.PET_SMOKE_DEVIL),
 	KRAKEN(NpcID.KRAKEN, 8400, ChronoUnit.MILLIS, ItemID.PET_KRAKEN),
 	KALPHITE_QUEEN(NpcID.KALPHITE_QUEEN_965, 30, ChronoUnit.SECONDS, ItemID.KALPHITE_PRINCESS),
-	DUSK(NpcID.DUSK_7889, 2, ChronoUnit.MINUTES, ItemID.NOON);
+	DUSK(NpcID.DUSK_7889, 2, ChronoUnit.MINUTES, ItemID.NOON),
+	ALCHEMICAL_HYDRA(NpcID.ALCHEMICAL_HYDRA_8622, 25200, ChronoUnit.MILLIS, ItemID.IKKLE_HYDRA_22750);
```

Comment:
```suggestion
	ALCHEMICAL_HYDRA(NpcID.ALCHEMICAL_HYDRA_8622, 25200, ChronoUnit.MILLIS, ItemID.IKKLE_HYDRA);
```
Use the base form of the pet for ItemID

Reply:
I found the base form harder to make out details for an icon, but it's pretty minor either way.

Reply:
Just thought it would be more consistent, since both corp and smoke devil also use the base version.